.github/workflows/release.yml

```diff
@@ -57,4 +57,16 @@ jobs:
           tag: ${{ env.TAG }}
           integration: 'nri-${{ env.INTEGRATION }}' # Required, with nri- prefix
           packageLocation: repo
-          upgrade: false
\ No newline at end of file
+          upgrade: false
+
+  notify-failure:
+    if: ${{ always() && failure() }}
+    needs: [publish-to-s3]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Notify failure via Slack
+        uses: archive/github-actions-slack@master
+        with:
+          slack-bot-user-oauth-access-token: ${{ secrets.COREINT_SLACK_TOKEN }}
+          slack-channel: ${{ secrets.COREINT_SLACK_CHANNEL }}
+          slack-text: "❌ `${{ env.REPO_FULL_NAME }}`: [release pipeline failed](${{ github.server_url }}/${{ env.ORIGINAL_REPO_NAME }}/actions/runs/${{ github.run_id }})."
```

Comment:
notice that in this case you are using two different identifiers REPO_FULL_NAME and ORIGINAL_REPO_NAME and the first one is not defined

Reply:
Updated: [6778e8d](https://github.com/newrelic/nri-postgresql/pull/106/commits/6778e8d6cf4a1bacd8703e7f7bf51ce063225fba), thanks!